src/frontend/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -40,7 +43,12 @@ export class VideoPlayer extends React.Component<
 
     if (video) {
       // Instantiate Plyr and keep the instance in state
-      this.setState({ player: new Plyr(this.videoNodeRef!) });
+      const player = this.props.createPlayer(
```

Comment:
directly assign this in the setState call, we don't have a use for the intermediate variable